homeassistant/components/zwave_js/__init__.py

```diff
@@ -190,21 +199,42 @@ def async_on_value_notification(notification: ValueNotification) -> None:
         )
 
     @callback
-    def async_on_notification(notification: Notification) -> None:
+    def async_on_notification(
+        notification: EntryControlNotification | NotificationNotification,
+    ) -> None:
         """Relay stateless notification events from Z-Wave nodes to hass."""
         device = dev_reg.async_get_device({get_device_id(client, notification.node)})
-        hass.bus.async_fire(
-            ZWAVE_JS_EVENT,
-            {
-                ATTR_TYPE: "notification",
-                ATTR_DOMAIN: DOMAIN,
-                ATTR_NODE_ID: notification.node.node_id,
-                ATTR_HOME_ID: client.driver.controller.home_id,
-                ATTR_DEVICE_ID: device.id,  # type: ignore
-                ATTR_LABEL: notification.notification_label,
-                ATTR_PARAMETERS: notification.parameters,
-            },
-        )
+        event_data = {
+            ATTR_TYPE: "notification",
```

Comment:
Then we'd need to rename this. Eg `"zwave_js_name"` or `"zwave_js_event_name"`.

Or split each event type on different Home Assistant event names, eg instead of having all be `zwave_js_event`, we have different: `zwave_js_notification`, `zwave_js_value_notification` etc.